huntsman_dust/util_fake_image.py

```diff
@@ -0,0 +1,144 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import huntsman_dust.utils as utils
+
+from collections import OrderedDict
+from scipy import fftpack
+from scipy.stats import powerlaw
+from astropy.table import Column
+from photutils.datasets import make_noise_image
+from photutils.datasets import make_random_models_table
+from photutils.datasets import make_gaussian_sources_image
+
+
+def fake_image(n_sources=100,
+               shape=(512, 512),
+               amplitude_r=[500, 1000],
+               std_dev=[1, 7],
+               random_state=77777,
+               gamma=-1.25,
+               noise=None,
```

Comment:
you might want to merge the noise parameters into a dictionary or array as it isn't clear if I need to provide all noise parameters. E.g. what if I give noise_mean, but not noise_stddev.

Reply:
Done